framework/include/wifi_manager/wifi_manager.h

```diff
@@ -133,8 +133,8 @@ typedef struct {
  */
 typedef struct {
 	char ssid[32];
-	uint16_t channel;
 	char passphrase[64];
+	uint16_t channel;
 } wifi_manager_softap_config_s;
```

Comment:
In SoftAP, ap_auth_type and ap_crypto_type are not set by user,
Why are AUTH_WPA2_PSK and CRYPTO_AES fixed?

Reply:
Because slsi API doesn't support other authencations. so I thought that confiuring authentication method of softap makes developers confuse.
If you think different then please share it with us.